features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -822,13 +822,13 @@
         <Resource context="(.*)/api/identity/entitlement/(.*)" secured="true" http-method="all">
             <Permissions>/permission/admin/manage/identity/pep</Permissions>
         </Resource>
-        <Resource context="(.*)/scim2/Users" secured="true" http-method="POST">
+        <Resource context="(.*)/scim2/Users(.*)" secured="true" http-method="POST">
             <Permissions>/permission/admin/manage/identity/usermgt/create</Permissions>
         </Resource>
         <Resource context="(.*)/scim2/Users" secured="true" http-method="GET">
```

Comment:
I think we need to do this for GET requests as well. (If it's not covered already)

Reply:
It's already covered for GET requests. Check following configuration which is already there. In that, there are two different permissions given in the two scenarios. 

       <Resource context="(.*)/scim2/Users" secured="true" http-method="GET">
<Permissions>/permission/admin/manage/identity/usermgt/list</Permissions>

        <Resource context="(.*)/scim2/Users/(.*)" secured="true" http-method="GET">
<Permissions>/permission/admin/manage/identity/usermgt/view</Permissions>

Reply:
So what about "(.*)/api/identity/consent-mgt/v1.0/consents" , "(.*)/api/identity/user/v1.0/me", etc .. ?

Reply:
For (.)/api/identity/consent-mgt/v1.0/consents, if we invoke with (.)/api/identity/consent-mgt/v1.0/consents/ and try to bypass the authentication/authorization, it will give following error. 
{"code":"CM_00036","message":"Bad Request","description":"PII Principal ID is required."}

I've added the same fix for scim2/Me endpoint.